app/models/kubernetes_image_scan.rb

```diff
@@ -0,0 +1,264 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'kubeclient'
+require 'securerandom'
+
+class KubernetesImageScan < Job
+  PROVIDER_CLASS = ManageIQ::Providers::Kubernetes::ContainerManager
```

Comment:
This class should be namespaced underneath `ManageIQ::Providers::Kubernetes::ContainerManager`